tenable/io/v3/was/attachments/api.py

```diff
@@ -0,0 +1,68 @@
+'''
+Attachments
+===========
+
+The following methods allow for interaction into the Tenable.io
+:devportal:`attachments <was-v2-attachments>` API.
+
+Methods available on ``tio.v3.was.attachments``:
+
+.. rst-class:: hide-signature
+.. autoclass:: AttachmentsAPI
+    :members:
+'''
+from io import BytesIO
+from typing import BinaryIO, Optional
+from uuid import UUID
+
+from tenable.io.v3.base.endpoints.explore import ExploreBaseEndpoint
+
+
+class AttachmentsAPI(ExploreBaseEndpoint):
+
+    _path = 'api/v3/was/attachments'
+
+    def download_attachment(self,
+                            attachment_id: UUID,
+                            fobj: Optional[BinaryIO] = None
+                            ) -> BinaryIO:
+        '''
+        Download an attachment.
+
+        :devportal:`attachments: download <was-v2-attachments-download>`
+
+        Args:
+            attachment_id:
+                The unique identifier for attachment.
+            fobj:
+                A file-like object to write the contents of the policy to.  If
```

Comment:
A file-like object to write the contents of the attachment to

Reply:
Done.